multimedia/minisatip/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=minisatip
+PKG_VERSION:=snapshot-20190710
```

Comment:
I wouldn't do it like this. There aren't any releases like this. I'd rather do it like this:

```
PKG_NAME:=minisatip
PKG_SOURCE_DATE:=2019-07-10
PKG_SOURCE_VERSION:=cd8ff66ad96be2e48ad8645a76ee7597f11477fe
PKG_RELEASE:=1
```